### PR TITLE
fclones: add page

### PR DESCRIPTION
**Pull request by MustCodeAl**
add tldr for fclones

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **0.34.0**

### CODE REVIEW

File: pages/common/fclones.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`fclones group --cache {{directory1}} {{directory2}}`
+
+- Search only the specified directory skipping the subdirectory and save the results into a file:
+
+`fclones group {{path/to/directory}} --depth 1 > dupes.txt`
+
+- Move the duplicates  in a `dupes.txt` to a different directory:
+
+`fclones move {{path/to/target_directory}} <dupes.txt`
```
Comment (kbdharun) [inline pages/common/fclones.md:20]:
```suggestion
- Search for duplicate files in the current directory:

`fclones group .`

- Search multiple directories for duplicate files and cache the results:

`fclones group --cache {{path/to/directory1 path/to/directory2}}`

- Search only the specified directory for duplicate files, skipping subdirectories and save the results into a file:

`fclones group {{path/to/directory}} --depth 1 > {{path/to/file.txt}}`

- Move the duplicate files in a TXT file to a different directory:

`fclones move {{path/to/target_directory}} < {{path/to/file.txt}}`
```

File: pages/common/fclones.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Perform a dry run for soft links  in a `dupes.txt` file without actually linking:
+
+`fclones link --soft <dupes.txt --dry-run 2>/dev/null`
```
Comment (kbdharun) [inline pages/common/fclones.md:24]:
```suggestion
- Perform a dry run for soft links  in a TXT file without actually linking:

`fclones link --soft < {{path/to/file.txt}} --dry-run 2 > /dev/null`
```

File: pages/common/fclones.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Delete the newest duplicates from the current directory without storing them in a file:
+
+`fclones group . | fclones remove --priority newest`
+
+- Preprocess jpg files in the current directory by using an external command  to strip their exif data before matching for duplicates:
```
Comment (kbdharun) [inline pages/common/fclones.md:30]:
```suggestion
- Preprocess JPG files in the current directory by using an external command  to strip their EXIF data before matching for duplicates:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by MustCodeAl** [comment]
Changes

> Thanks for your contribution. I have some suggestions for this page.

Changes accepted

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/fclones.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Search only the specified directory for duplicate files, skipping subdirectories and save the results into a file:
+
+`fclones group {{path/to/directory}} --depth 1 > {{path/to/file.txt}}`
+
+- Move the duplicate files in a TXT file to a different directory:
+
+`fclones move {{path/to/target_directory}} < {{path/to/file.txt}}`
+
+- Perform a dry run for soft links  in a TXT file without actually linking:
```
Comment (einverne) [inline pages/common/fclones.md:22]:
```suggestion
- Perform a dry run for soft links in a TXT file without actually linking:
```

File: pages/common/fclones.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`fclones link --soft < {{path/to/file.txt}} --dry-run 2 > /dev/null`
+
+- Delete the newest duplicates from the current directory without storing them in a file:
+
+`fclones group . | fclones remove --priority newest`
+
+- Preprocess JPEG files in the current directory by using an external command  to strip their EXIF data before matching for duplicates:
```
Comment (einverne) [inline pages/common/fclones.md:30]:
```suggestion
- Preprocess JPEG files in the current directory by using an external command to strip their EXIF data before matching for duplicates:
```

**Review by einverne — CHANGES_REQUESTED** [review]
hi @MustCodeAl  thank you for the new page, there are two extra spaces in the description.

**Review by einverne — APPROVED** [review]
Looks good now.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**